docs/application/dotnet/guides/nui/linear-layout.md

```diff
@@ -0,0 +1,128 @@
+# Linear Layout
+
+## Overview
+
+`LinearLayout` is a linear box layout, automatically handling children laid out vertically or horizontally. You can set the layout direction vertically or horizontally by using `LinearOrientation` property.
+
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/linearTotal.png">
+</td>
+</tr>
+</table>
+
+Positioning children in a linear form, one after the other, `CellPadding` can be provided to insert a space between each child. Unlike generic padding, this will not result in a space at the start, end or top, bottom of the layout.
+
+`LinearAlignment` defines where the children should start being positioned from, and it's very useful if the children do not use all the space of the parent.
+
+| Property               | Type            | Description  |
+| -----------------------| --------------- | ------------ |
+| `LinearOrientation`    | LinearLayout.Orientation | Gets/Sets Orientation of the linear layout : vertical or horizontal |
+| `LinearAlignment`      | LinearLayout.Alignment  | Gets/Sets the global child alignment to be used |
+| `CellPadding`          | Size2D      | Gets/Sets Spacing between the cells : horizontal space and vertical space |
+
+
+## Orientation
+
+A `LinearOrientation` indicates the direction of children layouts : horizontal or vertical. The default value is Horizontal.
+
+| Horizontal | Vetical |
+| -----------------------| --------------- |
+| ![Horizontal](./media/horizontalLayout.png) | ![Vertical](./media/verticalLayout.png)  |
+
+```csharp
+View layoutView = new View();
+var linearLayout = new LinearLayout();
+linearLayout.LinearOrientation = LinearLayout.Orientation.Horizontal;
+layoutView.Layout = layout;
+```
+
+
+## Alignment
+
+A `LinearAlignment` handles how Layout items are positioned within their parent Layout in the space. By default, items are aligned at the Top and Begin.
+
+You can choose from the below alignment options.
+
+| Alignment  | Info        |
+| ---------- | ----------- |
+| Begin      | At the left/right edge of the container (according to LTR/RTL direction for horizontal orientation) |
+| End        | At the right/left edge of the container (according to LTR/RTL direction for horizontal orientation) |
+| CenterHorizontal | At the horizontal center of the container (Vetical will be set to top) |
+| Top        | At the top edge of the container |
+| Bottom     | At the bottom edge of the container |
+| CenterVertical | At the vertical center of the container (Horizontal will be set to start) |
+| Center    | At both the vertical and horizontal center of the container |
+
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/linearAlignment.png">
+</td>
+</tr>
+</table>
```

Comment:
Global comment:

Kindly use MD format to display images. For more infomation, see: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#markdown-style-guide

Reply:
Done.